packages/manager/modules/nutanix/src/dashboard/nodes/node/general-info/constants.js

```diff
@@ -3,9 +3,10 @@ export const TRAVAUX_LINK = 'https://www.status-ovhcloud.com/';
 export const NUTANIX_NODE_STATUS = {
   DONE: 'DONE',
 };
-
+export const PRIVATE_BANDWIDTH_SERVICE_NODE_PREFIX = 'vrack-bandwidth';
```

Comment:
**question:** Dont we need to include `cluster-vrack-bandwidth`?

Reply:
Hello Anoop, For new catalog bandwidth plan codes have been renamed from "cluster-vrack-bandwidth" to "vrack-bandwidth".